tests/nightly/test_large_array.py

```diff
@@ -1199,6 +1199,15 @@ def test_full():
     assert a[-1][-1] == 3
 
 
+def test_gather():
+    arr = mx.nd.ones((LARGE_X, SMALL_Y))
+    idx = mx.nd.random.randint(0, LARGE_X, SMALL_X, dtype=np.int64)
```

Comment:
some comment here on the test will help.

Reply:
done. added test on large vector as well which i missed in previous commit